rust/arrow/src/compute/kernels/sort.rs

```diff
@@ -42,6 +42,8 @@ pub fn sort(values: &ArrayRef, options: Option<SortOptions>) -> Result<ArrayRef>
 }
 
 // sorts f32 in IEEE 754 total ordering
```

Comment:
Maybe "sorts" wasn't also the best way to describe it. Maybe `implements  comparison using IEEE 754 total ordering`